test/module/shared_model/backend_proto/proto_batch_test.cpp

```diff
@@ -191,3 +192,76 @@ TEST(TransactionBatchTest, BatchWithMissingSignatures) {
       << framework::expected::err(transaction_batch).value().error;
   ASSERT_FALSE(transaction_batch_val->value.hasAllSignatures());
 }
+
+/**
+ * Create test transaction builder
+ * @param acc_quorum - quorum number for setAccountDetail
+ * @param created_time - time of creation
+ * @param quorum - tx quorum number
+ * @return test tx builder
+ */
+inline auto makeTx(
```

Comment:
Maybe rename that into `makeTxBuilder` if it returns builder, but not transaction